client/my-sites/site-settings/section-export.jsx

```diff
@@ -1,88 +1,18 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import Immutable from 'immutable';
+import React, { Component } from 'react';
+import { Provider } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import CompactCard from 'components/card/compact';
-import Gridicon from 'components/gridicon';
-import Button from 'components/forms/form-button';
-import AdvancedOptions from 'my-sites/exporter/advanced-options';
+import ExporterContainer from 'my-sites/exporter';
 
-const defaults = {
-	advancedSettings: {
-		isVisible: false,
-		posts: {
-			isEnabled: true
-		},
-		pages: {
-			isEnabled: true
-		},
-		feedback: {
-			isEnabled: true
-		}
+export default class SiteSettingsExport extends Component {
+	render() {
+		return React.createElement( Provider, { store: this.props.store }, () => {
+			return <ExporterContainer />;
+		} );
```

Comment:
I'm curious why we are mixing JSX and plain-old-JS here… why not keep it all together?

```jsx
return (
	<Provider store={ this.props.store }>
		<ExporterContainer />
	</Provider>
);
```